CONTRIBUTING.md

```diff
@@ -5,15 +5,15 @@ While contribution guidelines are loosely defined, we recommend to read the foll
 
 ## Getting Started
 
-We use Yarn (and not NPM) to manage dependencies in stable and consistent ways. 
+We use pnpm (and not NPM) to manage dependencies in stable and consistent ways. 
 
-After installing [Yarn](https://yarnpkg.com/getting-started/install), clone this repository and run the following commands to install all dependencies and run the Gosling.js editor locally. We recommend you use Node 20. 
+After installing [pnpm](https://pnpm.io/installation), clone this repository and run the following commands to install all dependencies and run the Gosling.js editor locally. We recommend you use Node 20. 
 
 ```sh
 git clone https://github.com/gosling-lang/gosling.js.git # Clone the repository to your current directory
 cd gosling.js                                            # Navigate to gosling repository
-yarn                                                     # Install dependencies 
-yarn start                                               # Start a local server running the Gosling online editor 
+pnpm                                                     # Install dependencies 
```

Comment:
It looks like `pnpm` itself does not install packages

```suggestion
pnpm install                                             # Install dependencies 
```

Reply:
I just noticed this, too. Seems like it slipped through the cracks and wasn't fixed in the PR?

Reply:
Good catch, will make the PR now!